clients/cobol-lsp-vscode-extension/src/__tests__/PathUtilsTest.ts

```diff
@@ -0,0 +1,37 @@
+/*
```

Comment:
test failing because of the wrong path expected due to the OS

Reply:
updated